plugin/wavesurfer.spectrogram.js

```diff
@@ -73,10 +73,29 @@ WaveSurfer.Spectrogram = {
             this.container.removeChild(prevSpectrogram);
         }
 
+        var specLabels = (this.params.labels||false);
+        // if labels are active
+        if (specLabels) {
+            var specLabelsdiv = document.createElement('div');
+            specLabelsdiv.setAttribute('id', 'specLabels');
+            specLabelsdiv.style.cssText = 'position:absolute;left:0;z-index: 9;';
```

Comment:
You could use this method for a prettier CSS object: https://github.com/katspaugh/wavesurfer.js/blob/master/src/drawer.js#L98-L105